org.eclipse.jdt.core.tests.compiler/src/org/eclipse/jdt/core/tests/util/AbstractCompilerTest.java

```diff
@@ -55,6 +50,8 @@ public class AbstractCompilerTest extends TestCase {
 	public static final int F_20  = 0x20000;
 	public static final int F_21  = 0x40000;
 	public static final int F_22  = 0x80000;
+	/** Should be adopted if {@link CompilerOptions#getFirstSupportedJdkLevel()} changes */
+	public static final int FIRST_SUPPORTED_JAVA_VERSION = F_1_8;
```

Comment:
So why won't we call CompilerOptions#getFirstSupportedJdkLevel() and assign right away ?
 Nit: adapted. (sp)

Reply:
Because it is a completely different value & doesn't match the schema used here?